packages/itwin/grouping-mapping-widget/src/widget/components/GroupAction.tsx

```diff
@@ -179,68 +201,32 @@ const GroupAction = (props: GroupActionProps) => {
         "GroupingMappingWidget",
         iModelConnection,
       );
-      await props.goBack();
+      setDetails({
+        groupName: props.group?.groupName ?? "",
+        description: props.group?.description ?? "",
+      });
+      props.onSaveSuccess();
     } catch (error: any) {
       handleError(error.status);
+    } finally {
```

Comment:
This looks like some kind of bugfix. Maybe it would be better to pull out such things into their own separate PRs?